lib/MQTTPlateRecorder.js

```diff
@@ -241,6 +254,7 @@ export default class MQTTPlateRecorder extends PlateRecorder {
 			new URL(config.url),
 			config.baseTopic || DEFAULT_MQTT_BASE_TOPIC,
 			HassDiscovery.fromObject(config.hassDiscovery || {}),
+			config.enableImage || true,
```

Comment:
Please add a "DEFAULT_MQTT_PUBLISH_IMAGE" constant to constants.js instead of an inline boolean.